### DISCUSSION TITLE
Debugger is unavailable

### DISCUSSION THREAD

**Discussion by xyzzy2017** [TabularEditor/TabularEditor3#1023] [Ideas]
I used the Performance Analyzer in Power BI to capture the query of a slow visual.  I copied the query.  I pasted into TE 3.2.1. I clicked on Execute.  It ran fine (but slowly).

I right-clicked in a result in the Results pane.  Instead of 'Debug' appearing as a pop-up menu, it displayed 'Debug Unavailable' or something close to that.  The measure is locally defined.  Is that not yet supported?

Thanks

**Comment by otykier** [comment]
Local/implicit measures are not yet supported: https://docs.tabulareditor.com/te3/features/dax-debugger.html#limitations-and-known-issues

**Comment by xyzzy2017** [comment]
Thank you Daniel.
